BabyPatterns/FeedingsVM.swift

```diff
@@ -1,11 +1,76 @@
 import Foundation
 import Framework_BabyPatterns
 import Library
+import WatchConnectivity
 
-class FeedingsVM: Loggable {
+extension FeedingsVM: WCSessionDelegate {
+    public func session(_: WCSession, activationDidCompleteWith _: WCSessionActivationState, error: Error?) {
+        print("Completed activation: \(error?.localizedDescription ?? "n/a error")")
+    }
+
+    public func sessionDidBecomeInactive(_: WCSession) {
+        print("session did become inactive")
+    }
+
+    public func sessionDidDeactivate(_: WCSession) {
+        print("session did deactivate")
+    }
+
+    public func session(_: WCSession, didReceiveMessage _: [String: Any]) {
+        print("Received message!")
+    }
+
+    public func session(_: WCSession, didReceiveMessage message: [String: Any], replyHandler: @escaping ([String: Any]) -> Void) {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/jasonzurita/BabyPatterns/blob/9a1d22d8ad71ad19efd2b877aa3b663fe9f8e3a5/BabyPatterns/FeedingsVM.swift#L23"></span>Line should be 120 characters or less: currently 130 characters<br />
<code>line_length</code> <code>FeedingsVM.swift:23</code></td>
    </tr>
  </tbody>
</table>